### PR TITLE
[15.0][FIX] mail_activity_team: warning Two fields have same label

### PR DESCRIPTION
**Pull request by dsolanki-initos**
Test case failed from **microsoft_outlook_single_tenant** module

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Review by etobella — APPROVED** [review]

File: mail_activity_team/models/mail_activity.py [hunk 1]
```diff
@@ -20,8 +20,10 @@ def _get_default_team_id(self, user_id=None):
             )
         return self.env["mail.activity.team"].search(domain, limit=1)
 
-    user_id = fields.Many2one(required=False)
-    team_user_id = fields.Many2one(related="user_id", readonly=False)
+    user_id = fields.Many2one(string="user", required=False)
+    team_user_id = fields.Many2one(
+        string="team user", related="user_id", readonly=False
+    )
```
Comment (pedrobaeza) [inline mail_activity_team/models/mail_activity.py]:
```suggestion
    user_id = fields.Many2one(string="User", required=False)
    team_user_id = fields.Many2one(
        string="Team user", related="user_id", readonly=False
    )
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Sorry, I didn't see that all was lower, and you can rebase now that the fix is merged.

File: mail_activity_team/models/mail_activity.py (hunk 1 above)
Comment (dsolanki-initos) [reply mail_activity_team/models/mail_activity.py]: @pedrobaeza , I have updated as per your feedback. Kindly check it.

**Comment by pedrobaeza** [comment]
Please squash both commits into one.

**Review by hieulucky111 — APPROVED** [review]

**Comment by dsolanki-initos** [comment]
> Please squash both commits into one.

@pedrobaeza , I have squashed the commits.

**Review by pedrobaeza — APPROVED** [review]
Thanks. Next time, for this simple amends, do `git commit --amend` directly and `git push -f` for avoiding hammering the CI system.

/ocabot merge minor

**Pull request merged**